examples/index.js

```diff
@@ -4,15 +4,15 @@ const FA = require('react-fontawesome')
 
 // Spinning
 ReactDOM.render(
-  <FA name='twitter' spin />,
+  <FA name="twitter" spin />,
   document.getElementById('spin')
 )
 
 // Stack
 ReactDOM.render(
-  <span className='fa-stack fa-lg'>
-    <FA name='square-o' stack='2x' />
-    <FA name='twitter' stack='1x' />
+  <span className="fa-stack fa-lg">
+    <FA name="square-o" stack="2x" />
```

Comment:
Please keep consistent with the styling of the app an use single quotes everywhere, thanks!